code/index.html

```diff
@@ -26,14 +26,24 @@ <h1>Does the person have</h1>
         <optgroup label="hair">
           <option value="brown">brown hair</option>
           <option value="yellow">yellow hair</option>
+          <option value="grey">grey hair</option>
+          <option value="black">black hair</option>
+          <option value="orange">orange hair</option>
+          <option value="purple">purple hair</option>
+          <option value="white">white hair</option>
+          <option value="hidden">hidden</option>
           <!-- Add the rest of hair colors as options  -->
         </optgroup>
         <optgroup label="eyes">
           <option value="hidden">hidden eyes</option>
+          <option value="blue">blue eyes</option>
+          <option value="brown">brown eyes</option>
+          <option value="green">green eyes</option>
           <!-- Add the rest of eye colors as options  -->
         </optgroup>
         <optgroup label="accessories">
           <option value="glasses">glasses</option>
+          <option value="hat">hat</option>
           <!-- Add the other accessory option here. (hat)  -->
         </optgroup>
         <optgroup label="other">
```

Comment:
Could be an idea to add more options for the "other" category. I played the game a few times and more than once you got to a point where you couldn't ask anymore questions but had to guess who still. Just an idea if you were to develop it further. :)